ui/src/dashboards/components/CellEditorOverlay.js

```diff
@@ -492,6 +561,9 @@ class CellEditorOverlay extends Component {
       (!!query.measurement && !!query.database && !!query.fields.length) ||
       !!query.rawText
 
+    const visualizationColors =
+      cellWorkingType === 'gauge' ? gaugeColors : singleStatColors
+
```

Comment:
ahh I like that idea a lot

Reply:
Looked into this change in more detail -- looks like it might cause some problems. If the `cellWorkingType` becomes anything other than `gauge` or `singlestat` then it would throw an error and I'd have to guard against it. Could end up being more code than what is currently here